### PR TITLE
[lora] Add lora configs and validation

### PR DESCRIPTION
**Pull request by xyang16**
## Description ##

Brief description of what this PR is about

- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: tests/integration/llm/prepare.py [hunk 1]
```diff
@@ -538,6 +542,10 @@
         "true",
         "option.max_lora_rank":
         64,
+        "option.fully_sharded_loras":
+        "true",
+        "option.long_lora_scaling_factors":
+        "(4.0, )",
```
Comment (sindhuvahinis) [inline tests/integration/llm/prepare.py:548]: Why does it have to be a strict tuple? Can we get comma separated strings and convert into tuple internally? This is what we do for context_length_estimate for tnx.

Comment (xyang16) [reply tests/integration/llm/prepare.py]: Changed. Now we can get comma separated strings and convert into tuple.

File: engines/python/setup/djl_python/properties_manager/vllm_rb_properties.py [hunk 2]
```diff
@@ -51,7 +52,7 @@ class VllmRbProperties(Properties):
     max_lora_rank: Optional[int] = 16
     fully_sharded_loras: bool = False
     lora_extra_vocab_size: int = 256
-    long_lora_scaling_factors: Optional[Tuple[float]] = None
+    long_lora_scaling_factors: Optional[Tuple[float, ...]] = None
```
Comment (sindhuvahinis) [inline engines/python/setup/djl_python/properties_manager/vllm_rb_properties.py:55]: QQ, `...` Is this to allow empty tuple to this?

**Review by sindhuvahinis — APPROVED** [review]

File: engines/python/setup/djl_python/properties_manager/vllm_rb_properties.py (hunk 2 above)
Comment (xyang16) [reply engines/python/setup/djl_python/properties_manager/vllm_rb_properties.py:55]: ... allow for tuple with variable length. for example, (3.0,) (3.0,4.0)

Tuple[float] only allow for one element in the tuple.

**Review by siddvenk — APPROVED** [review]

**Pull request merged**